nntrainer/tensor/manager.cpp

```diff
@@ -287,24 +341,58 @@ void Manager::untrackLayerInOuts(const std::string layer_name) {
  * @brief Initialize the inputs/outputs for the layer
  */
 void Manager::initializeInOuts(bool trainable) {
+  initialize(trainable);
```

Comment:
this is wrong
this will reset the weights which have been loaded from the saved file.

Reply:
resolved.